api.planx.uk/admin/session/html.ts

```diff
@@ -1,8 +1,10 @@
 import { generateApplicationHTML } from "@opensystemslab/planx-core";
 import { $admin } from "../../client";
-import type { NextFunction, Request, Response } from "express";
+import type { RequestHandler } from "express";
 import type { PlanXExportData } from "@opensystemslab/planx-core/types";
 
+type HTMLExportHandler = RequestHandler<{ sessionId: string }, string>;
```

Comment:
This is a neat thing I just learned we can do with Express types

You can do `RequestHandler<PathParams, ResponseBody, RequestBody, QueryParams>`